src/app/shared/components/accordion-control/accordion-control.component.scss

```diff
@@ -1,3 +1,11 @@
 .govuk-accordion__controls {
   width: 100%;
 }
+
+.flex {
+  display: flex;
+}
+
+.padding {
+  padding: 0 15px;
```

Comment:
is there a gov uk "flexible" padding?

Reply:
[Sure is!](https://design-system.service.gov.uk/styles/spacing/)